.reuse/dep5

```diff
@@ -0,0 +1,7 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Contact: Digital Bazaar, Inc. <support@digitalbazaar.com>
+
+Files: *.md .gitignore .npmrc package.json test/.npmrc test/mocha/.eslintrc
+    test/package.json .editorconfig .github/* test/static/*.html
```

Comment:
> * Is there a wildcard format that would be easier?

The danger of `Files: *` is that it would apply to anything in the future also--which may not be the case (as with the logo you added to the core bedrock repo). So, being explicit (or somewhat explicit as with `*.md`) seems better...if annoying at times.

The [reuse helper tool](https://git.fsfe.org/reuse/tool#reuse) does a good job of highlighting what's been missed, and those can be easily added to this list or in a new section for whatever license they're supposed to be under.

> * Does the reuse linter check this for extra files?  The `.eslintrc` files are `.eslingrc.cjs` in this project and have license text.  Don't need to be in this list.

So, when things appear in both list the reuse helper tool does throw an error. We can certainly be more consistent here.

> * How is a reader supposed to know to look at this obscure file for license info?  If going all in on this meta info, is best practice to use direct comments and `.license` files where practical?

Reader's of the REUSE spec would know/care to look, and we could explain that a bit in the readme's or an "OSS Policy" repo we could/should make.

I chose not to go with the `.license` file option because of the "clutter" and concerns around needing to avoid them when bundling, etc. Happy to discuss using those instead, but I found a verbose `.reuse/dep5` to be preferable.

Reply:
I had not heard of the REUSE spec until you started using it here.  I suspect that is the case for most people.  And none of them will look at a random `.reuse` file since we are numb to tons of dot files we can ignore in every repo.  Would you you look there for license info?  I think it needs to be clear in docs, or somewhere, if a magic file is the official source of license info.  It's also strange the dep5 file doesn't cover all the files, like I think it would in Debian.

Reply:
They're actually moving to a `reuse.yaml` file, but that won't be much different for your concern. We can and should explain our OSS policy/process/compliance and mention it in the README's. However, any/all of this is going beyond (deliberately) the same statement/claim made in the LICENSE file--which people are familiar with.

The REUSE specification is one of many options for being thorough and explicit in stating ownership and licensing and it's one of the few that I've found that provides this capability along with the code--many others require storing claims elsewhere in a compliance database of some kind.

REUSE is standardized, has an active community, and is in use. It also fully supports creating SPDX Software Bill of Materials (SBOMs)--which is the real goal here.

Hopefully that explains the choice more. Happy to chat if you still have concerns.